client/src/pages/about.js

```diff
@@ -0,0 +1,403 @@
+/* eslint-disable max-len */
+import React, { Fragment } from 'react';
+import Helmet from 'react-helmet';
+import { Grid, Col, Row, Table } from '@freecodecamp/react-bootstrap';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import {
+  faLinkedin,
+  faGithub,
+  faTwitter,
+  faYoutube,
+  faMedium,
+  faFacebook
+} from '@fortawesome/free-brands-svg-icons';
+import {
+  faLock,
+  faBalanceScale,
+  faUniversity,
+  faBook
+} from '@fortawesome/free-solid-svg-icons';
+
+import { Link, Spacer } from '../components/helpers';
+
+import './about.css';
+
+const AboutPage = () => {
+  return (
+    <Fragment>
+      <Helmet title='About the freeCodeCamp community | freeCodeCamp.org' />
+      <Spacer />
+      <Grid className='container'>
+        <Row>
+          <Col
+            className='questions'
+            md={6}
+            mdOffset={3}
+            sm={10}
+            smOffset={1}
+            xs={12}
+            >
+            <h2 className='text-center'>Frequently Asked Questions</h2>
+            <hr />
+            <h4>What is freeCodeCamp?</h4>
+            <p>
+              We’re a community that helps you learn to code, then get
+              experience by contributing to open source projects used by
+              nonprofits.
+            </p>
+            <h4>How can you help me learn to code?</h4>
+            <p>
+              You'll learn to code by completing coding challenges and
+              building projects. You'll also earn verified certifications
+              along the way. We also encourage you to join a study group in
+              your city so you can code in-person with other people.
+            </p>
+            <h4>Is freeCodeCamp really free?</h4>
+            <p>Yes. Every aspect of freeCodeCamp is 100% free.</p>
+            <h4>
+              Can freeCodeCamp help me get a job as a software developer?
+            </h4>
+            <p>
+              Yes. Every year, thousands of people who join the freeCodeCamp
+              community get their first software developer job. If you're
+              curious, you can{' '}
+              <Link
+                external={true}
```

Comment:
@raisedadead No need to set `external={true}` for absolute links

Reply:
See [this diff for what we are upto](https://github.com/freeCodeCamp/freeCodeCamp/pull/35243/files/7e827a27744d1edced2e6e2cb051dc429dbb076a#diff-8cc13ecb83eae1c4cc561432bcfd97d9), I borrowed this from our footer logic and moved to a helper util.

Reply:
Most of the time we would want to open links in new tabs, other than our own pages. Sometimes our own pages may suite better to be opened in new tabs.

This is just to unify some of it.

This commit explains the use-case: https://github.com/freeCodeCamp/freeCodeCamp/pull/35243/commits/7e827a27744d1edced2e6e2cb051dc429dbb076a

Reply:
Yes, I saw it.

Reply:
> Most of the time we would want to open links in new tabs, other than our own pages. Sometimes our own pages may suite better to be opened in new tabs.

Yes, and `external` in this component is used for making local links behave as external.

Reply:
Oh Fish! Right, I misunderstood what you meant. Thanks, will fix it.